CHANGELOG.md

```diff
@@ -12,8 +12,13 @@
   - `podOverrides`
   - `affinity`
 
+### Fixed
```

Comment:
I'd be in favor of removing the changelog, as this operator is not using S3 at all

Reply:
done